### PR TITLE
Update downloadcmd section

### PR DESCRIPTION
**Pull request by MIZwally**

### CODE REVIEW

File: docs/prepare.md [hunk 1]
```diff
@@ -43,14 +43,23 @@ The contents of `~/.config/python_keyring/keyringrc.cfg` should be:
...
 
-If the correct password is not returned, then running `keyring.set_password("nda-tools", "<username>", "<password>")` should fix the issue.
+If the correct password is not returned, then run the following to fix the NDA credentials issue (replacing `<username>` with your actual NDA username and `<password>` with your actual NDA password):
+
+```shell
+cd ~/abcd-fasttrack2bids
+poetry run python -c 'keyring.set_password("nda-tools", "<username>", "<password>")'
+```
+
+If you are encountering the following error upon running this command 'ModuleNotFoundError: No module named 'keyrings'', then it is likely that your versions of 'keyring' and 'keyring.alt' are out of date. Make sure that the 'keyring' and 'keyrings.alt' versions respectively are '23.13.1' and '3.1' or greater.
```
Comment (ericearl) [inline docs/prepare.md]:
```suggestion
If you are encountering the following error upon running this command, then it is likely that your versions of 'keyring' and 'keyring.alt' are out of date. Make sure you have installed python poetry from the README.md installation instructions and that the 'keyring' and 'keyrings.alt' versions respectively are '23.13.1' and '3.1' or greater.

`ModuleNotFoundError: No module named 'keyrings'`
```

Comment (ericearl) [reply docs/prepare.md]: How's that?

File: docs/prepare.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ cd ~/abcd-fasttrack2bids
 poetry run python -c 'keyring.set_password("nda-tools", "<username>", "<password>")'
 ```
 
-If you are encountering the following error upon running this command 'ModuleNotFoundError: No module named 'keyrings'', then it is likely that your versions of 'keyring' and 'keyring.alt' are out of date. Make sure that the 'keyring' and 'keyrings.alt' versions respectively are '23.13.1' and '3.1' or greater.
+If you are encountering the following error upon running this command, then it is likely that your versions of 'keyring' and 'keyring.alt' are out of date. Make sure you have installed python poetry from the README.md installation instructions and that the 'keyring' and 'keyrings.alt' versions respectively are '23.13.1' and '3.1' or greater.
```
Comment (MIZwally) [inline docs/prepare.md]:
```suggestion
If you are encountering the following error upon running this command, then it is likely that your versions of `keyring` and `keyring.alt` are out of date. Make sure you have installed python poetry from the README.md installation instructions and that the `keyring` and `keyrings.alt` versions respectively are `23.13.1` and `3.1` or greater.
```

File: docs/prepare.md [hunk 3]
```diff
@@ -43,14 +43,25 @@ The contents of `~/.config/python_keyring/keyringrc.cfg` should be:
...
 
 ```shell
-python3
-import keyring
-keyring.get_password("nda-tools", "<username>")
+cd
+git clone https://github.com/nimh-dsst/abcd-fasttrack2bids.git
+cd ~/abcd-fasttrack2bids
+poetry install
+poetry run python -c 'import keyring ; keyring.get_password("nda-tools", "<username>")'
```
Comment (ericearl) [inline docs/prepare.md]:
```suggestion
poetry run python -c 'import keyring ; print(keyring.get_password("nda-tools", "<username>"))'
```

**Review by ericearl — APPROVED** [review]
lgtm!

**Pull request merged**